examples/rtfm-demo/Cargo.toml

```diff
@@ -9,6 +9,11 @@ cortex-m-rtfm = "0.4"
 panic-semihosting = "0.5"
 cortex-m-semihosting = "0.3"
 
+[dependencies.nrf52810-hal]
+version = "0.1"
```

Comment:
I suggest just starting with version 0.6, to match the other crates.

Reply:
Done